.github/workflows/version.yml

```diff
@@ -27,8 +27,10 @@ jobs:
           git checkout master
           git config user.name OSBotify
 
-      - name: Checkout new branch
-        run: git checkout -b version-bump-${{ github.sha }}
+      - name: Create new branch
+        run: |
+          git checkout -b version-bump-${{ github.sha }}
+          git push --set-upstream origin version-bump-${{ github.sha }}
```

Comment:
We need to push so that the branch exists on the remote, but we don't need to change anything or commit before pushing, because the create-pull-request action basically harvests any changes and/or commits present locally on the runner but not on the remote and uses them to create a pull request.

So we make our changes, commit them (so that we can create a tag and push it), then run the create-pull-request action, and it will use the local commit to create the pull request.